components/panel.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: 'Panel'
+description: 'Specify the content of the right side panel'
+icon: 'panel-right'
+---
+
+You can use the `<Panel>` component to customize the right side panel of a page. By default, the right side panel displays a table of contents listing any titles and subtitles.
```

Comment:
The Panel is actually built to replace the table of contents. It doesn't "have" the table of contents by default. Would mention that instead.